pkg/workloads/cortex/downloader/download.py

```diff
@@ -13,24 +13,35 @@
 # limitations under the License.
 
 import argparse
+import os
+import base64
+import json
 
+from cortex.lib import util
 from cortex.lib.storage import S3
 from cortex.lib.log import get_logger
 
 logger = get_logger()
 
 
 def start(args):
-    bucket_name, prefix = S3.deconstruct_s3_path(args.download_from)
-    s3_client = S3(bucket_name, client_config={})
-    s3_client.download(prefix, args.download_to)
+    download = json.loads(base64.urlsafe_b64decode(args.download))
+    for download_arg in download:
+        from_path = download_arg["from"]
+        to_path = download_arg["to"]
+        bucket_name, prefix = S3.deconstruct_s3_path(from_path)
+        s3_client = S3(bucket_name, client_config={})
+        s3_client.download(prefix, to_path)
+        if download_arg["unzip"] and os.path.basename(from_path).endswith("zip"):
```

Comment:
nit: I think we can drop the `and os.path.basename(from_path).endswith("zip")` since if the caller passed in `unzip=True` we can assume they know what they're doing